os/arch/arm/src/common/up_createstack.c

```diff
@@ -185,7 +189,12 @@ int up_create_stack(FAR struct tcb_s *tcb, size_t stack_size, uint8_t ttype)
 #endif
 		{
 			/* Use the user-space allocator if this is a task or pthread */
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION	/* address must be aligned to 32 byte for MPU region */
+			stack_size = stack_size + CONFIG_MPU_STACK_GUARD_SIZE;
```

Comment:
`stack_size` is a param variable, We'd better declare one local variable.

Reply:
yes, used local variable for all the references